src/test/html-transform_test.ts

```diff
@@ -297,6 +297,22 @@ suite('htmlTransform', () => {
       assertEqualIgnoringWhitespace(replaceGiantScripts(result), expected);
     });
 
+    test('does not add AMD loader when no modules', () => {
+      const input = `
+        <html><head></head><body>
+          <script>console.log('non-module');</script>
+          <script src="depA.js"></script>
+        </body></html>`;
+
+      const result = htmlTransform(input, {
+        injectAmdLoader: true,
+        js: {
+          transformEsModulesToAmd: true,
+        },
+      });
+      assertEqualIgnoringWhitespace(replaceGiantScripts(result), input);
```

Comment:
Do we need to call `replaceGiantScripts` here?

Reply:
Done.